framework/src/modules/chain/blocks/block_v1.js

```diff
@@ -30,6 +30,9 @@ const BigNum = require('@liskhq/bignum');
 const { validator } = require('@liskhq/lisk-validator');
 const { validateTransactions } = require('../transactions');
 const blockVersion = require('./block_version');
+
+// TODO: Due to UNKNOWN issue the require for utils returned empty object
+delete require.cache[require.resolve('./utils')];
```

Comment:
This is wrong. We shouldn't try to find a hack to resolve this thing. The problem occurs due to circular dependency introduced by utils.js file depending on block.js file which in turn depends on blockV1.js and blockV2.js and they import utils.js again.
A simpler fix would be to duplicate code and declare sortTransactions function within blockV1 and blockV2. I think it makes sense since it is possible that different versions of blocks sort transactions differently.

Reply:
Ignorre this comment, since it was fixed in the later commit.

Reply:
Its already resolved.

Reply:
Is this not removed in one of the other commits?

Reply:
Yes its removed, this change here is outdated.